spec/features/user_can_view_past_orders_spec.rb

```diff
@@ -1,33 +1,62 @@
 require 'rails_helper'
 
 RSpec.feature "user can view past orders" do
-  scenario "sees index of orders" do
-    pending
-    db_repo = FactoryJordan.new
-    user = db_repo.create_users(1)[0]
+  scenario "through the order show page" do
+    user = User.create(
+      first_name: "Jordan",
+      last_name: "Lawler",
+      username: "jlawler",
+      password: "password"
+    )
+
+    ApplicationController.any_instance.stub(:current_user).and_return(user)
+
+    order = user.orders.create(
+      status: "paid",
+      total_price: 12,
+      created_at: Time.new(2016, 01, 01),
+      updated_at: Time.new(2016, 01, 02),
```

Comment:
Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead.